cdc/processor/pipeline/sink.go

```diff
@@ -103,16 +104,37 @@ func (n *sinkNode) getCheckpointTs() model.ResolvedTs {
 // no more events can be sent to this sink node afterwards.
 func (n *sinkNode) stop(ctx context.Context) (err error) {
 	// table stopped state must be set after underlying sink is closed
-	defer n.state.Store(TableStateStopped)
-	return n.closeTableSink(ctx)
+	defer n.state.Store(tablepb.TableStateStopped)
+	if n.sinkV1 != nil {
+		err = n.sinkV1.Close(ctx)
+		if err != nil {
+			return
+		}
+		log.Info("sinkV1 is closed",
+			zap.Int64("tableID", n.tableID),
+			zap.String("namespace", n.changefeed.Namespace),
```

Comment:
Can we implement the `fmt.Stringer` interface for the `Changefeed` to avoid we will add `namespace` and `changefeed-name` everywhere.

For example:
```go
func (c *Changefeed) String() string {
    return fmt.Srpintf("%s:%s", c.Namesapce, c.ID)
}


log.Info("something", zap.Stringer("changefeed", n.changefeed))
```

Reply:
Yes, we can. Let's do it in another PR.